pysb/simulator/bng.py

```diff
@@ -152,7 +153,8 @@ def run(self, tspan=None, initials=None, param_values=None, n_runs=1,
                               verbose=verbose_bool,
                               output_dir=output_dir,
                               output_prefix=output_file_basename,
-                              cleanup=cleanup,
+                              cleanup=self.cleanup if cleanup is None
```

Comment:
The formatting here is a little unfortunate. Maybe just normalize the value of `cleanup` with `if cleanup is None: cleanup = self.cleanup` up with all the other parameter checking?

Reply:
Sounds good. Done.